### ISSUE TITLE
What is the recommended resolution of the asset to create the splash screen?

### ISSUE THREAD

**Issue by alejandrogiubel** [henriquearthur/flutter_native_splash#89]
I have a png image with 5141 x 7705. When I create the splash screen with this plugin my app's splash screen is a very very 
big image. How can I fix this?

**Comment by Amerr** [comment]
yes we need documentation for the perfect image size for the splash screen

**Comment by NLeone** [comment]
I also join the request, please, I would like to know how to include a brand logo centered on a black background, I can do that, but I need it to look the same in most devices, at least portrait. If anyone has an idea how to do it, we would appreciate it as it is a great complement.

**Comment by itmediacreations** [comment]
I'm using logo with resolution 1024x1024 on small devices it takes almost all the screen.

**Comment by xalikoutis** [comment]
400x400 i think is optimal

**Comment by jonbhanson** [comment]
Thanks everyone for your suggested resolutions.  I'm not sure there is a "perfect" resolutions, but it is helpful to hear about what others are using.
My logo works pretty well at 1300x350.

**Comment by joj3000** [comment]
This guy made a video on this particular package. Maybe it'll solve your issues :
https://www.youtube.com/watch?v=8ME8Czqc-Oc
(At the end of the video he talks about the full screen native splash)

**Comment by postpasspost** [comment]
I used 640*194.
It's too small on Android and big on iOS. Is there a way to scale it up a little bit on Android?

**Comment by roslamir** [comment]
I think if you make your image with an aspect ratio of 9x16 you should be okay.

**Comment by Dubemtopsite** [comment]
@alejandrogiubel Were you able to later solve the issue

**Comment by dJani97** [comment]
It would be nice to add a parameter to somehow constrain the image. I had to edit my splash screen image and add a considerable amount of transparent padding around the edges to make it look good.

**Comment by Dubemtopsite** [comment]
@dJani97 Thanks 

I was able to resolve mine by reducing the image size to 450px x 450px

**Comment by Faiizii** [comment]
512 x 512 or 1024 x 1024 is good resolution for app icon. Because, when we generate the native splash by using this library it creates splash icon of dimension 192x192 (max on android) and 1024x1024 (max on iOS). Therefore, if we use the too large assets, it will increase the build size whereas more than 1024 is not needed (in mobile phone splash). If we use too small (any value less than 512) it will pixelate on large devices like Tabs/iPads.

**Comment by raian-ruku** [comment]
for android_12, i did this

```
# App icon without an icon background: This should be 1152×1152 pixels, and fit within a circle
    # 768 pixels in diameter.  To fit a 1152x1152 image within a circle with a 768 diameter, simply 
    # ensure that the most important design elements of your image are placed within a circular area
    # with a 768 diameter at the center of the 1152x1152 canvas.
```

i copied this portion and gave it to chatgpt with the image. it asked to generated the image according to this and i permitted. used the image and it was perfectly rendered.
